packages/link/react.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="react" />
```

Comment:
what does `types=react` control?

Reply:
My understanding is that this references the types declared in the `@types/react`, making it so I can use `React.Component` down below.